admin_manual/configuration/files/file_sharing_configuration.rst

```diff
@@ -72,6 +72,27 @@ Configure your sharing policy on your Admin page in the Sharing section.
    `Wrong folder date when syncing 
    <https://github.com/owncloud/core/issues/7009>`_ for discussion of this.
 
+Blacklist Groups From Receiving Shares
+--------------------------------------
+
+Sometimes it's necessary or desirable to block groups from receiving shares.
+For example, if a group has a significant number of users (> 5,000) or if it's a system group, then it can be advisable to block them from receiving shares.
+
+In these cases, ownCloud administrators can blacklist one or more groups, so that they do not receive shares.
+To blacklist one or more groups via the Web UI, under "**Admin -> Settings -> Sharing**", add one or more groups, comma-separated to the "Files Sharing" list.
```

Comment:
In the UI you do not type commas. It lets you select groups from a dropdown list and adds them.

Reply:
Thanks. I had issues yesterday that prevented me confirming.